src/server/routes/controllers/Middlewares/checkSession.js

```diff
@@ -0,0 +1,15 @@
+const passport = require('passport')
+const strategyUtils = require('../../../passport/strategyUtils')
+
+function userAuthentication(req, res, next) {
+	if(req.isAuthenticated())
+		next()
+	else {
+		const loginStrategyHandler = strategyUtils.strategyHandlers(req, res, next).loginStrategyHandler
+		const authenticate = passport.authenticate(strategyUtils.STRATEGY_NAME, loginStrategyHandler) 
```

Comment:
We're back to baked flour problem :stuck_out_tongue: 
authenticate returns authenticate.

Reply:
ok